src/hotspot/share/aot/aotCodeHeap.cpp

```diff
@@ -188,10 +188,6 @@ void AOTLib::verify_config() {
   verify_flag(_config->_enableContended, EnableContended, "EnableContended");
   verify_flag(_config->_restrictContended, RestrictContended, "RestrictContended");
 
-  if (!TieredCompilation && _config->_tieredAOT) {
-    handle_config_error("Shared file %s error: Expected to run with tiered compilation on", _name);
-  }
-
```

Comment:
It's always running with the tiered policy now. Prior to this change if you attempted to run an AOT library with tiered-style profiling compiled in with the non-tiered policy it wouldn't work correctly. That's why we had this check. Now tiered is the only policy, so it will work correctly whether you have the AOT code with profiling or not.

Reply:
Okay. Got it